enhancements/ingress/gateway-api-crd-life-cycle-management.md

```diff
@@ -0,0 +1,470 @@
+The latter situation has more complexities. We'll refer to this process as "CRD
+Management Succession", and cover it's implications and logic below.
+
+#### CRD Management Succession
+
+Taking over the management of the Gateway API CRDs from a previous entity
+can be turmultuous as we will require:
+
+* no other actors managing the CRDs from here on out
+* only standard CRDs can be deployed (no experimental)
+* the CRDs to be deployed at an exact version/schema
+* OR the CRDs not to be present at all
+
+We will require this by providing a pre-upgrade check in the previous release
+that verifies these are true and sets `upgradable=false` if any of them are not.
+
+> **Note**: Upstream Gateway API unfortunately layered experimental versions of
+> CRDs on top of the same GVK as the standard ones, so unfortunately for initial
+> release users will be unable to use experimental as we can't deliver that as
+> a part of our supported API surface. It will be feasible to add a gate to
+> enable experimental later, but because of this layering this will not be a
+> great user experience and will require the cluster to become tainted. We are
+> tracking and supporting [an effort] in upstream Gateway API to separate
+> experimental into its own group, which we expect to help move towards a better
+> overall experience for users who want experimental Gateway API features going
+> forward.
+
+**We simply can not anticipate all of the negative effects** succession will
+have on existing implementations if the cluster admin forces through the upgrade
+check. As an extra precaution for users forcing upgrades precipitously, we will
+provide an admin gate to both provide an extra warning and gather consent from
+the cluster admin to take over the management of the CRDs. This will be
+accompanied by detailed documentation on what the admin should check and how to
+fufill the checks, which will also be linked from the description provided in
+the admin gate.
+
+The admin will be responsible for ensuring the safety of succession. If they
+force through the pre-upgrade check AND the admin gate, the admin gate will be
+left behind to help aid investigation of the cause of problems when the upgrade
+goes wrong.
+
+[an effort]:https://github.com/kubernetes-sigs/gateway-api/discussions/3497
+
+### Workflow Description
+
+> Explain how the user will use the feature. Be detailed and explicit.  Describe
+> all of the actors, their roles, and the APIs or interfaces involved. Define a
+> starting state and then list the steps that the user would need to go through to
+> trigger the feature described in the enhancement. Optionally add a
+> [mermaid](https://github.com/mermaid-js/mermaid#readme) sequence diagram.
+>
+> Use sub-sections to explain variations, such as for error handling,
+> failure recovery, or alternative outcomes.
+
+**cluster-admin** is a human user responsible for managing a cluster.
+
+1. Start with a 4.18 cluster with conflicting CRDs.
+2. Upgrade to 4.19.
```

Comment:
I suppose that we should acknowledge the admin gate before upgrading.

Reply:
This section was also out-of-date, and from a previous iteration. Updated in 50fcb989d70734ff36e8f78bb88ee9eac31d2dcf LMKWYT?